sbpack/lib.py

```diff
@@ -71,7 +72,8 @@ def normalize_to_map(obj: Union[list, dict], key_field: str):
         raise RuntimeError("Expecting a dictionary or a list here")
 
 
-def normalize_to_list(obj: Union[list, dict], key_field: str, value_field: str):
+def normalize_to_list(
+        obj: Union[list, dict], key_field: str, value_field: Union[str, None]):
```

Comment:
Maybe better Optional[str] instead of Union[str, None].

Reply:
Changed it in the latest commit :)